### ISSUE TITLE
Add common footer with a Github icon and link to this repo

### ISSUE THREAD

**Issue by shanzzam** [stellar/stellar-demo-wallet#141]
### What problem does your feature solve?
Better visibility to code and a way for users to provide feedback, log bugs, and submit changes themselves.

### What would you like to see?
A footer added to the mainpage with the Github icon and link to the [Issues](https://github.com/stellar/stellar-demo-wallet/issues) or [Main](https://github.com/stellar/stellar-demo-wallet) page of this repo

### What alternatives are there?
We can keep the announcement banner at the top of the page for a bit but other than that there isn't an easy way to discover this repo.

**Comment by shanzzam** [comment]
Text on the footer: "Check out our Github repo more information and to log issues"
That may be too wordy @lydiat

**Comment by quietbits** [comment]
@shanzzam @lydiat We could show a longer description on hover.

![image](https://user-images.githubusercontent.com/7346473/119143814-0ae8ae00-ba16-11eb-8b11-7efa2dde4afa.png)

P.S. I will add the GitHub icon once we've finalized the link text.

**Comment by lydiat** [comment]
![Screen Shot 2021-05-24 at 4 08 18 PM](https://user-images.githubusercontent.com/1917909/119417844-99c22880-bcab-11eb-9188-5991a31942a2.png)

**Comment by quietbits** [comment]
More links added in https://github.com/stellar/stellar-demo-wallet/pull/171

**Comment by shanzzam** [comment]
@quietbits looks good! I've reviewed and will close this as complete.
